### DISCUSSION TITLE
Is there a way to disable most-visited sites section on the new tab page?

### DISCUSSION THREAD

**Discussion by kagamihamai** [bromite/bromite#1979] [Ideas]
I've dug through the settings, no such setting could be found.

I wonder if there is a toggle in the flags or something like that, or this feature simply doesn't exist?

I have to manually delete most-visited sites shortcuts every time if I don't want to just clear all browsing history (which I don't).

I do understand that I can set custom homepage and enable it for new tabs as well, but I'll lose the *Bookmarks* and *Downloads* entries, which are not present on an incognito new tab.

So enable history in incognito plus enable always open links in incognito tabs isn't really working as smoothly as a simple, ordinary new tab.

Is it possible to just disable most-visited sites on a new tab? (For better privacy, and prevent accidental touches.)

**Comment by csagan5** [comment]
Feature was previously asked here: #436
Something similar used to be present in Bromite until https://github.com/bromite/bromite/issues/116

Correct way to do this would be to disable the site engagement as explained here: https://github.com/bromite/bromite/issues/1942#issuecomment-1094254223

**Reply by kagamihamai** [reply]
Thank you for replying.

I've looked up the sources you pointed. But as far as I can tell, the flags regarding the site engagement setting is no longer available in the newer versions of Chromium?

**Reply by csagan5** [reply]
Yes, someone would need to create a patch for it.

**Reply by kagamihamai** [reply]
Got it, thank you for the explanation.
